### DISCUSSION TITLE
Need sample!!

### DISCUSSION THREAD

**Discussion by GRNHACKER** [hfhbd/routing-compose#255] [Q&A] [answered]
Can you show me some sample using this library for compose web?

**Comment by hfhbd** [comment, marked as answer]
The Readme contains a sample. 
The linked (but ugly) homepage uses this library: https://hfhbd.github.io/routing-compose/, sources: https://github.com/hfhbd/routing-compose/tree/main/integrationTest
Or see https://github.com/hfhbd/ComposeTodo live at https://todo.softwork.app
